template.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
+
+insert_into_file 'config/application.rb', after: /config\.load_defaults [0-9.]+$/ do
+  "\n    config.add_autoload_paths_to_load_path = true"
+end
+
```

Comment:
This autoload config needs to be setup here before attempting to load the hyrax gem and install generator.